plugins/network-elements/nsx/src/main/java/org/apache/cloudstack/utils/NsxControllerUtils.java

```diff
@@ -45,6 +45,10 @@ public static String getNsxNatRuleId(long domainId, long accountId, long dataCen
         return String.format("D%s-A%s-Z%s-%s%s-NAT", domainId, accountId, dataCenterId, resourcePrefix, resourceId);
     }
 
+    public static String getNsxDistributedFirewallPolicyRuleId(String segmentName, long ruleId) {
+        return String.format("%s-P%s", segmentName, ruleId);
```

Comment:
I think we decided to use R to identify rules

Reply:
Fixed, thanks